grafana/perf/regression-summary.json

```diff
@@ -98,129 +363,240 @@
             ]
           }
         },
-        "overrides": [
-          {
-            "matcher": {
-              "id": "byName",
-              "options": "server_version.buildDate"
-            },
-            "properties": [
-              {
-                "id": "custom.width",
-                "value": 167
-              }
-            ]
-          },
-          {
-            "matcher": {
-              "id": "byName",
-              "options": "run_host"
-            },
-            "properties": [
-              {
-                "id": "custom.width",
-                "value": 210
-              }
-            ]
-          }
-        ]
+        "overrides": []
       },
       "gridPos": {
-        "h": 8,
+        "h": 6,
         "w": 24,
         "x": 0,
         "y": 5
       },
-      "id": 142,
+      "id": 171,
+      "interval": "1d",
+      "links": [
+        {
+          "targetBlank": true,
+          "title": "OCP version",
+          "url": "https://openshift-release.apps.ci.l2s4.p1.openshiftapps.com/"
+        }
+      ],
       "options": {
-        "footer": {
-          "fields": "",
-          "reducer": [
-            "sum"
-          ],
-          "show": false
+        "alignValue": "right",
+        "legend": {
+          "displayMode": "hidden",
+          "placement": "bottom"
         },
-        "showHeader": true,
-        "sortBy": []
+        "mergeValues": true,
+        "rowHeight": 0.9,
+        "showValue": "always",
+        "tooltip": {
+          "mode": "single"
+        }
       },
       "pluginVersion": "8.4.0-pre",
       "targets": [
         {
-          "alias": "",
-          "bucketAggs": [],
+          "alias": "ocp",
+          "bucketAggs": [
+            {
+              "field": "timestamp",
+              "id": "2",
+              "settings": {
+                "interval": "auto"
+              },
+              "type": "date_histogram"
+            }
+          ],
           "datasource": {
             "type": "elasticsearch",
-            "uid": "eh85eR3nz"
+            "uid": "mg2W2WD7z"
```

Comment:
What does this refer to?

Reply:
its internal Grafana uuid, all the json generate automatically.